src/main/kotlin/exercise2/task5/CreateUserDSL.kt

```diff
@@ -49,11 +49,16 @@ internal data class Address(
  */
 
 internal fun user(initUser: User.() -> Unit): User {
-    TODO("Implement me!!!")
+    val usr = User()
```

Comment:
Bravo! Dobro ste shvatili kako se pravi DSL. :)

Ovaj kod može da se napiše i sa `apply` scope funkcijom na sledeći način:
```kotlin
return User().apply(initUser)
```